### PR TITLE
test(tables): e2e tests

### PR DESCRIPTION
**Pull request by ZaymonFC**

### CODE REVIEW

File: packages/apps/composer-app/src/playwright/tables.spec.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+import { expect, type Page, test } from '@playwright/test';
+
+import { faker } from '@dxos/random';
+
+import { AppManager } from './app-manager';
+
+faker.seed(0);
+
+const table = {
```
Comment (wittjosiah) [inline packages/apps/composer-app/src/playwright/tables.spec.ts:13]:
```suggestion
const Table = {
```

File: packages/apps/composer-app/src/playwright/tables.spec.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    await expect(table.getHeaderCell(host.page)).toHaveCount(3);
+
+    const newColumnName = faker.lorem.word();
+
+    await table.getColumnMenu(host.page).last().click();
+    await table.getOpenColumnSettings(host.page).click();
+    await table.getColumnSettingsLabel(host.page).fill(newColumnName);
+    await table.getColumnSettingsSave(host.page).click();
+
+    expect(await table.getHeaderCell(host.page).nth(1).textContent()).toBe(newColumnName);
```
Comment (wittjosiah) [inline packages/apps/composer-app/src/playwright/tables.spec.ts:171]:
```suggestion
    await expect(table.getHeaderCell(host.page).nth(1)).toHaveText(newColumnName);
```

**Review by wittjosiah — COMMENTED** [review]
I generally use `test` prefix for PRs on e2e tests

File: packages/apps/composer-app/src/playwright/tables.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ZaymonFC) [reply packages/apps/composer-app/src/playwright/tables.spec.ts:13]: 👍

File: packages/apps/composer-app/src/playwright/tables.spec.ts [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    await expect(table.getDataRow(host.page)).toHaveCount(1);
+
+    const ROWS = 5;
+    for (let i = 0; i < ROWS; i++) {
+      const cell = table.getDataRow(host.page).last().getByTestId('table.cell').first();
+      await cell.click();
+      await host.page.keyboard.type('test');
+      await host.page.keyboard.press('Enter');
+      // sleep for a while to make sure the row is added
+      await host.page.waitForTimeout(100);
```
[the remark this answers is not in the document]
Comment (ZaymonFC) [reply packages/apps/composer-app/src/playwright/tables.spec.ts:97]: Yes, we need to wait for the add row to be populated. I guess I could get tricker and wait for the correct `nth` instead.

**Pull request merged**